flask/json.py

```diff
@@ -222,8 +222,8 @@ def get_current_user():
             "id": 42
         }
 
-    For security reasons only objects are supported toplevel.  For more
-    information about this, have a look at :ref:`json-security`.
+    Support for serializing top-level arrays was added in Flask 1.0. For
```

Comment:
This should use a [`versionchanged`](http://sphinx-doc.org/markup/para.html#directive-versionadded) block.